paper.bib

```diff
@@ -130,7 +130,7 @@ @article{lysakovski2021development
 
 @article{castle2022,
   author={Castle, James R and Duan, Jingwei and Feng, Xue and Chen, Quan},
-  title={Development of a virtual source model for Monte Carlo-based independent dose calculation for varian linac},
+  title={Development of a virtual source model for {Monte} {Carlo}-based independent dose calculation for varian linac},
```

Comment:
At first I thought that this lower case varian was in need of capitalisation and curly braces... but that is actually how it reads in the publication!

Reply:
Haha, well there you go!